network/domain.go

```diff
@@ -46,6 +46,7 @@ func GetClusterDomainName() string {
 	once.Do(func() {
 		f, err := os.Open(resolverFileName)
 		if err != nil {
+			domainName = defaultDomainName
```

Comment:
This seems like not doing anything since this is set by default above, or am I missing something? In which case would this be necessary?